11ty/definitions/guys-gals.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Guys and Gals
+slug: guys-gals
+defined: false
+speech: noun
+reading:
+  - text: Hey Guys
+    href: https://heyguys.cc/
+alt_words:
+  - everyone
+  - folks
+  - all
+  - friends
+  - team
+  - y'all
+  - yinz
+  - people
+  - peeps
+  - comrades
+---
+
+"Hey guys" is a common salutation, but it's not gender neutral. People who aren't men can feel excluded when addressed as "guys," even if that is not the intent.
```

Comment:
Typically for our definitions we start with what the definition means, so this could be like: "colloquial for a person (guy, singular) or group of people (guys, plural); its origins come from Guy Fawkes."

I'd recommend adding this resource to further reading:
https://www.washingtonpost.com/news/volokh-conspiracy/wp/2015/05/14/the-origin-of-the-word-guy/
https://www.etymonline.com/word/guy

Adding a bit around the impact. Dude is a good model for this; you could copy a lot of the language over: https://www.selfdefined.app/definitions/dude/

Reply:
Regarding the impact: Can I defer this to someone else?
Since I'm part of the privileged class (white, male, studied etc), I don't feel like being the right person to assess the impact.